confluent_kafka_helpers/consumer.py

```diff
@@ -107,3 +109,61 @@ def _get_topics(self, config):
     @property
     def is_auto_commit(self):
         return self.config.get('enable.auto.commit', True)
+
+
+class AvroRawConsumer(Consumer):
```

Comment:
We can use the `AvroConsumer` for this case, but then we need to override the `poll` method and also call the `Consumer.poll` directly:

```python
class Consumer:
    def poll(self, timeout):
        print("Consumer poll")
        return {'answer': 42}


class AvroConsumer(Consumer):
    def poll(self, timeout=None):
        print("AvroConsumer poll")
        message = super().poll(timeout)

        print("AvroConsumer decoding message")
        return {'answer': 42, 'decoded': True}


class AvroLazyConsumer(AvroConsumer):
    
    def poll(self, timeout=None):
        print("AvroLazyConsumer poll")
        message = Consumer.poll(self, timeout)
        return message
``` 

In this way we will remove the need of copy&paste the `__init__`, but we will need to mix the `AvroConsumer and Consumer` in the same class and this can be confusing.


The other way won't work, if we just inherit from `AvroConsumer` and call the `super().poll` we will have the same performance issues.

Reply:
@simonwahlgren